scripts/refresh_token.py

```diff
@@ -1,24 +1,22 @@
 #!/usr/bin/env python2.7
```

Comment:
probably needs to be updated!

Reply:
Agreed, but not in scope of this linting